apps/root/src/pages/transfer/components/transfer-form/index.tsx

```diff
@@ -99,7 +99,12 @@ const TransferForm = () => {
   const trackEvent = useTrackEvent();
   const { token: selectedToken, recipient, amount } = useTransferState();
   const selectedNetwork = useSelectedNetwork();
-  const tokenParam = useToken(tokenParamAddress, false, false, Number(chainIdParam));
+  const tokenParam = useToken({
+    tokenAddress: tokenParamAddress,
+    checkForSymbol: true,
+    filterForDca: false,
+    chainId: selectedNetwork.chainId,
```

Comment:
should this be Number(chainIdParam)?

Reply:
Same response from above!

Reply:
This would then work just on a second re-render right? cant we parse the network before so its already set here, and then do the dispatches?

Reply:
Did some changes around this! and also some token-resetting when changing the network